config/gradle/jre.gradle

```diff
@@ -15,14 +15,14 @@
  */
 
 // Uses Bellsoft Liberica JRE
-def jreVersion = '8u212'
-def jreUrlBase = "https://download.bell-sw.com/java/${jreVersion}/bellsoft-jre${jreVersion}"
+// https://bell-sw.com/pages/liberica_install_guide-11.0.7/
+def jdkVersion = '11.0.7+10' // jre version haven't jmod (needed for jlink/jpackager)
 def jreUrlFilenames = [
-        Linux64   : 'linux-amd64.tar.gz',
-        Linux32   : 'linux-i586.tar.gz',
-        Windows64 : 'windows-amd64.zip',
-        Windows32 : 'windows-i586.zip',
-        Mac       : 'macos-amd64.zip'
+        Linux64   : 'linux-amd64-full.tar.gz',
+//        Linux32   : 'linux-i586-full.tar.gz', // -full builds not available for linux-i586
```

Comment:
One more reason to drop 32-bit support... we should adjust the matrix build setup then to remove Linux32 from there: https://github.com/MovingBlocks/TerasologyLauncher/blob/master/.github/workflows/asset-upload.yml#L14